data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```diff
@@ -318,11 +333,22 @@
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "NO_BREATHE" ]
   },
+  {
+    "id": "mon_cracker_lbass",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_medium",
```

Comment:
You need to change the species so that it's not a fish and use one of the sweet species. It's important because that's part of the system that make candy monster hostile to candy players but not to regular players.